### ISSUE TITLE
AvCheckbox - have a checked prop

### ISSUE THREAD

**Issue by diegodesouza** [Availity/availity-reactstrap-validation#190]
Right now I'm able to set a `checked={checked}` prop except that I get this warning. 

``` 
react_devtools_backend.js:6 Warning: A component is changing an uncontrolled input of type checkbox to be controlled. Input elements should not switch from uncontrolled to controlled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://fb.me/react-controlled-components
```

Wouldnt the `checked` prop be checked by default if set to true initially? where as the  `defaultChecked` wont retain its value after i change it later, as in a toggle event.

**Comment by diegodesouza** [comment]
Is this something we can do?

**Comment by nylon22** [comment]
can you link a code sandbox demonstrating this issue?

**Comment by diegodesouza** [comment]
@nylon22 check it out [here](https://stackblitz.com/edit/react-availity-testing-tcrlae)

**Comment by diegodesouza** [comment]
Not an issue anymore
